src/Packetery/Module/Plugin.php

```diff
@@ -406,12 +406,13 @@ static function () {
 		add_action( 'woocommerce_email_footer', [ $this, 'renderEmailFooter' ] );
 		add_filter( 'woocommerce_shipping_methods', array( $this, 'add_shipping_method' ) );
 
+		$orderListScreenId = 'woocommerce_page_wc-orders';
 		add_filter( 'views_edit-shop_order', [ $this->gridExtender, 'addFilterLinks' ] );
+		add_filter( "views_{$orderListScreenId}", [ $this->gridExtender, 'addFilterLinks' ] );
```

Comment:
Složené závorky jsou tady navíc. Vyřešil bych to pomocí sprintf, aby to bylo jednotné.